src/test/mir-opt/uninhabited_fallthrough_elimination.eliminate_fallthrough.UninhabitedEnumBranching.diff

```diff
@@ -0,0 +1,92 @@
+- // MIR for `eliminate_fallthrough` before UninhabitedEnumBranching
```

Comment:
It looks like the source code of the new test cases is not committed? (We should probably check for stale MIR opt files in compiletest)

Reply:
Yeah, no idea how I managed to do that, but fixed now